### DISCUSSION TITLE
How can i focus on the current buffer when opening nvim-tree?

### DISCUSSION THREAD

**Discussion by samyarkd** [nvim-tree/nvim-tree.lua#2815] [Q&A] [answered]
I have mapped `<leader>e` for opening nvim-tree with `NvimTreeToggle` but there is another command called `NvimTreeFindFile` which will focus on the opened buffer in the NvimTree. i'm new to Nvim and i don't know how can i d it in a way when i press `<leader>e` it automatically focus on the current buffer

**Comment by gegoune** [comment, marked as answer]
Have a look at `:h nvim-tree.update_focused_file.enable`. Setting it to `true` in your config might give you result you are after.

**Reply by samyarkd** [reply]
great thanks, it's exactly what i was looking for
